.test-infra/jenkins/job_LoadTests_ParDo_Flink_Python.groovy

```diff
@@ -79,29 +79,31 @@ def batchScenarios = { datasetName -> [
                         environment_type     : 'DOCKER',
                 ]
         ],
-        [
-                title          : 'ParDo Python Load test: 20M 100 byte records 200 times',
-                test           : 'apache_beam.testing.load_tests.pardo_test',
-                runner         : CommonTestProperties.Runner.PORTABLE,
-                pipelineOptions: [
-                        job_name             : 'load-tests-python-flink-batch-pardo-2-' + now,
-                        project              : 'apache-beam-testing',
-                        publish_to_big_query : true,
-                        metrics_dataset      : datasetName,
-                        metrics_table        : 'python_flink_batch_pardo_2',
-                        influx_measurement   : 'python_batch_pardo_2',
-                        input_options        : '\'{' +
-                                '"num_records": 20000000,' +
-                                '"key_size": 10,' +
-                                '"value_size": 90}\'',
-                        iterations           : 200,
-                        number_of_counter_operations: 0,
-                        number_of_counters   : 0,
-                        parallelism          : 5,
-                        job_endpoint         : 'localhost:8099',
-                        environment_type     : 'DOCKER',
-                ]
-        ],
+// TODO(BEAM-10270): Takes too long time to execute (currently more than 2 hours). Re-enable
```

Comment:
@dmvk Since you are the batch expert, do you have any idea about this?

Reply:
As pointed out in the bug, there was sharp increase in `python_dataflow_batch_pardo_2_runtime` on May 7: http://metrics.beam.apache.org/d/MOi-kf3Zk/pardo-load-tests?orgId=1&var-processingType=batch&var-sdk=python&from=1584703436990&to=1592475836990&panelId=3&fullscreen

Perhaps this PR is related? https://github.com/apache/beam/pull/11558

Reply:
Also the comment should say "more than 4 hours"

Reply:
It could be related but it doesn't appear that any changes were made to the batch mode.

Reply:
> Also the comment should say "more than 4 hours"

From https://ci-beam.apache.org/job/beam_LoadTests_Python_ParDo_Flink_Batch/270/console:

```
16:21:33 INFO:apache_beam.runners.portability.portable_runner:Job state changed to RUNNING
19:48:20 Build timed out (after 240 minutes). Marking the build as aborted.
```

That gives us slightly more than 3 hours. The rest of time is spent on Pardo test with 10 iterations. I'll change the comment to "more than 3 hours"

Reply:
@mxm Sorry, back from vacation.

Unfortunatelly I'm not really familiar with the portable runner. I guess that some of the recent  "legacy" batch runner changes may have affected the portable runner too, but it's hard to say.

Can you `git bisect` to a commit that introduces the regression?